R/pkg/R/DataFrame.R

```diff
@@ -2732,13 +2732,24 @@ setMethod("union",
             dataFrame(unioned)
           })
 
-#' Return a new SparkDataFrame containing the union of rows
-#'
-#' This is an alias for `union`.
+#' Return a new SparkDataFrame containing the union of rows.
+#' This is an alias for \code{union}.
```

Comment:
actually, we do need a newline - L2735 is the title as this is a new page. could you build the roxygen2 doc to double check?

Reply:
You are right, it is the title and newline is required. I manually checked.